examples/dataflow-streaming-benchmark/src/main/java/com/google/cloud/pso/pipeline/StreamingBenchmark.java

```diff
@@ -194,24 +260,31 @@ public void setup() throws IOException {
         }
 
         schema = byteArrayOutputStream.toString();
+        this.validateSchema(dataGenerator, schema);
```

Comment:
One of the benefits of allowing invalid schema is that it allows you to test how your pipeline handles malformed data. Do you think we should put this functionality behind a pipeline option or is the expectation that the user should cover that in their unit tests?

Reply:
Since the message needs to be deserialized to extract the values for the special fields schema must be valid.  Added a pipelineoption called validateSchema that can be set to false to test pipeline with invalid schema.  However in such cases attributes will not be populated.